<html><head><meta content="text/html; charset=UTF-8" http-equiv="content-type"><style type="text/css">@import url('https://themes.googleusercontent.com/fonts/css?kit=wAPX1HepqA24RkYW1AuHYA');ol{margin:0;padding:0}table td,table th{padding:0}.c2{border-right-style:solid;padding:2pt 2pt 2pt 2pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:bottom;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:58.5pt;border-top-color:#000000;border-bottom-style:solid}.c4{color:#000000;font-weight:400;text-decoration:none;vertical-align:baseline;font-size:11pt;font-family:"Arial";font-style:normal}.c19{padding-top:0pt;padding-bottom:0pt;line-height:1.15;orphans:2;widows:2;text-align:center}.c0{padding-top:0pt;padding-bottom:0pt;line-height:1.15;orphans:2;widows:2;text-align:left}.c11{padding-top:0pt;padding-bottom:0pt;line-height:2.0;orphans:2;widows:2;text-align:center}.c5{padding-top:0pt;padding-bottom:0pt;line-height:2.0;orphans:2;widows:2;text-align:left}.c20{background-color:#ffffff;-webkit-text-decoration-skip:none;color:#1155cc;text-decoration:underline;text-decoration-skip-ink:none}.c8{background-color:#ffffff;color:#222222;text-decoration:none;vertical-align:baseline;font-style:normal}.c10{padding-top:0pt;padding-bottom:0pt;line-height:1.15;text-align:right}.c25{border-spacing:0;border-collapse:collapse;margin-right:auto}.c7{color:#000000;text-decoration:none;vertical-align:baseline;font-style:normal}.c22{padding-top:0pt;padding-bottom:16pt;line-height:1.15;text-align:left}.c16{padding-top:0pt;padding-bottom:0pt;line-height:1.0;text-align:center}.c14{padding-top:0pt;padding-bottom:0pt;line-height:1.15;text-align:left}.c1{font-size:12pt;font-family:"Times New Roman";font-weight:400}.c13{font-weight:400;font-size:10pt;font-family:"Arial"}.c9{font-weight:700;font-size:12pt;font-family:"Times New Roman"}.c18{font-weight:700;font-size:14pt;font-family:"Times New Roman"}.c24{background-color:#ffffff;max-width:468pt;padding:72pt 72pt 72pt 72pt}.c3{font-weight:400;font-family:"Calibri"}.c21{color:inherit;text-decoration:inherit}.c23{background-color:#ffffff;color:#222222}.c6{height:11pt}.c15{height:16.5pt}.c12{text-indent:36pt}.c17{font-style:italic}.c26{vertical-align:sub}.title{padding-top:0pt;color:#000000;font-size:26pt;padding-bottom:3pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}.subtitle{padding-top:0pt;color:#666666;font-size:15pt;padding-bottom:16pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}li{color:#000000;font-size:11pt;font-family:"Arial"}p{margin:0;color:#000000;font-size:11pt;font-family:"Arial"}h1{padding-top:20pt;color:#000000;font-size:20pt;padding-bottom:6pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h2{padding-top:18pt;color:#000000;font-size:16pt;padding-bottom:6pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h3{padding-top:16pt;color:#434343;font-size:14pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h4{padding-top:14pt;color:#666666;font-size:12pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h5{padding-top:12pt;color:#666666;font-size:11pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h6{padding-top:12pt;color:#666666;font-size:11pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;font-style:italic;orphans:2;widows:2;text-align:left}</style></head><body class="c24"><p class="c5 c6"><span class="c7 c1"></span></p><p class="c5 c6"><span class="c7 c1"></span></p><p class="c5 c6"><span class="c7 c1"></span></p><p class="c5 c6"><span class="c7 c1"></span></p><p class="c5 c6"><span class="c7 c1"></span></p><p class="c16 c6"><span class="c7 c18"></span></p><p class="c16 c6"><span class="c7 c18"></span></p><p class="c16 c6"><span class="c7 c18"></span></p><p class="c16 c6"><span class="c7 c18"></span></p><p class="c16 c6"><span class="c7 c18"></span></p><p class="c6 c16"><span class="c7 c18"></span></p><p class="c16 c6"><span class="c7 c18"></span></p><p class="c16 c6"><span class="c7 c18"></span></p><p class="c16"><span class="c7 c18">Differential Gene Expression Analysis of Ulcerative Colitis Tissue Types</span></p><p class="c16"><span class="c7 c18">Chandershekhar Shori, Steven Kim</span></p><p class="c16"><span class="c7 c18">BCH 339N--Dr. Cenik</span></p><p class="c16"><span class="c7 c18">7 May 2020</span></p><p class="c5 c6"><span class="c7 c9"></span></p><p class="c5 c6"><span class="c7 c9"></span></p><p class="c5 c6"><span class="c7 c9"></span></p><p class="c5 c6"><span class="c7 c9"></span></p><p class="c5 c6"><span class="c7 c9"></span></p><p class="c5 c6"><span class="c7 c9"></span></p><p class="c5 c6"><span class="c7 c9"></span></p><p class="c5 c6"><span class="c7 c9"></span></p><p class="c5 c6"><span class="c7 c9"></span></p><p class="c5 c6"><span class="c7 c9"></span></p><p class="c5 c6"><span class="c7 c9"></span></p><p class="c5"><span class="c7 c9">Abstract:</span></p><p class="c5 c12"><span class="c7 c1">Inflammatory bowel disease (IBD) has plagued many individuals across the world by manifesting itself primarily in the forms of Crohn&rsquo;s disease and ulcerative colitis. &nbsp;Currently these potentially fatal illnesses have no cure, and to be diagnosed with these illnesses means to battle IBD for the rest of your lifetime. Additionally, not much is known about which tissue specifically causes this IBD phenotype. However, research has shown that the key to ameliorate these illnesses may lie in differential gene expression. In this investigation, we looked at oligonucleotide microarray data from 202 ulcerative colitis patients in order to determine the differential gene expression among whether the individual has ulcerative colitis, region of the colon, and the interaction between both of these variables. We identified 4 genes that have significant interaction between ulcerative colitis phenotype and the sigmoid region.</span></p><p class="c5 c6 c12"><span class="c7 c1"></span></p><p class="c5"><span class="c7 c9">Background:</span></p><p class="c5 c12"><span class="c1 c7">Inflammatory bowel disease (IBD) is a chronic autoimmune illness that causes inflammation throughout one&rsquo;s gastrointestinal tract and has been known to be the leading cause of colon cancer. Symptoms include diarrhea, constipation, stomach pain, and rectal bleeding. There are two primary types of IBD: ulcerative colitis and Crohn&rsquo;s disease. &nbsp;Ulcerative colitis causes inflammation that is limited to the colon, while Crohn&rsquo;s disease causes inflammation that is spread out through the entire gastrointestinal tract. Currently there is no cure for these severe illnesses, but both of these illnesses seem to be intermittently linked due to the fact that both cause inflammation in various parts of the digestive symptom and manifest through similar symptoms. &nbsp;It has been hypothesized that certain genes may lead to a genetic predisposition for IBD that may make it more likely for a patient to develop this illness in their lifetime. The reason being is that mutations in genes can lead to upregulation and downregulation of the production of certain proteins that lead to a potentially exacerbated response in times when immune action is not needed. This exacerbated response is what is known as an autoimmune response as the body begins to hurt itself by causing excessive inflammation when the body is at rest. In the long run, if this response is not contained, it can lead to permanent scarring in the intestines. This scarring can decrease nutrient and water absorption, which can lead to abnormal growth as the body is not able to absorb the enough resources to provide to its cells. Furthermore, in extreme cases, the inability to absorb nutrients can lead to other bodily organs to malfunction or stop working entirely, which can result in the death of an individual. That is why it is necessary to find a way to contain these illnesses in order to help people in the long run, and the solution to this may lie in gene expression. </span></p><p class="c5 c6"><span class="c7 c1"></span></p><p class="c5"><span class="c7 c9">Data:</span></p><p class="c5 c12"><span class="c1">For this investigation, a dataset from the </span><span class="c1 c23">Gene Expression Omnibus (GEO) database will be used (GDS 3268). This dataset contains information from 202 individuals, with 135 of those individuals having ulcerative colitis (UC) in different locations of the colon. The dataset contains information about how expression of different genes varies for patients who have UC affecting different parts of their colon. The oligonucleotide microarray data was normalized to the log2 ratio. The source of the microarray is from Agilent Technologies and used to identify genes (</span><span class="c1 c20"><a class="c21" href="https://www.google.com/url?q=https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc%3DGPL1708&amp;sa=D&amp;source=editors&amp;ust=1612590061833000&amp;usg=AOvVaw32XEjjmgC6VSsTFjTX1MMK">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GPL1708</a></span><span class="c8 c1">).</span></p><p class="c5 c6"><span class="c8 c1"></span></p><p class="c5"><span class="c7 c9">Goals:</span></p><p class="c5 c12"><span class="c1 c8">The goal of this study is to contribute knowledge to the field of IBD by showing which genes show a significant role in the development of ulcerative colitis and vary in their expression in different parts of the colon.</span></p><p class="c5"><span class="c1 c23">&nbsp;</span></p><p class="c5"><span class="c9">Methods:</span></p><p class="c5 c12"><span class="c7 c1">First, import the GEO file using the &ldquo;GEOquery&rdquo; package. Then, &nbsp;we shall remove all the genes that do not contain values in all the columns and are denoted by &ldquo;NA.&rdquo; The reason being is that these genes represent incomplete cases when we are in need of gene expression data for that gene for all 202 patients that are being investigated in this study. If these &ldquo;NA&rdquo; values were kept, then an error is observed in the code. </span></p><p class="c5 c12"><span class="c1">After confirming that the data was normalized and in log2 ratio, a linear model with </span><span class="c1 c17">&beta;</span><span class="c1 c17 c26">0 </span><span class="c1">= 0 </span><span class="c7 c1">was created using the limma package. Data was inputted as a 2x2 factorial matrix with condition normal and condition ulcerative colitis as one variable and sigmoidal/non-sigmoidal tissue as the other. For the purposes of this model, any region that was not sigmoidal (ascending colon, descending colon, terminal ileum) were grouped into one. As such, there were 4 levels to this linear model. Contrasts were made to compare variables to each other as well as the interaction, the effect of the condition phenotype on the regional expression. We are most particularly interested in the interaction as this indicates that the ulcerative condition phenotype will impact specifically the sigmoidal region. The contrast matrix corresponds with coefficients of the data. A value&rsquo;s coefficient corresponds with which gene has a higher fold expression relative to the contrasted variable. This is shown in Table 1.</span></p><p class="c5 c6 c12"><span class="c7 c1"></span></p><p class="c5"><span class="c1 c17">Table 1: </span><span class="c7 c1">Coefficients for each contrast, UC stands for ulcerative colitis, wt stands for the normal phenotype relative to ulcerative colitis, sigmoid means tissue samples taken from the sigmoidal tissue/region, and other means non-sigmoidal tissue. </span></p><p class="c5"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 98.67px;"><img alt="" src="images/image5.png" style="width: 624.00px; height: 98.67px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5 c12"><span class="c7 c1">With this model, t-statistics and f-statistics were calculated between each of these contrasts. We set a threshold for differentially expressed genes (p&lt;0.001) and looked at genes with the most significance, or lowest p-value as well as larger coefficients. These differentially expressed genes are given as log2 fold changes. </span></p><p class="c5 c6"><span class="c7 c9"></span></p><p class="c5"><span class="c7 c9">Preliminary Analysis:</span></p><p class="c5 c12"><span class="c7 c1">Data is shown to be normalized among all trials. This is shown in Figure 1. Since the data is normalized and no uneven distributions exist, a linear model is created and corresponding tests were calculated. The significant genes from the contrast are shown below in Table 2.</span></p><hr style="page-break-before:always;display:none;"><p class="c5 c6"><span class="c7 c1"></span></p><p class="c5"><span class="c1 c17">Figure 1:</span><span class="c7 c1">&nbsp;Distribution of normalized gene density from GEO database record GDS3268</span></p><p class="c11"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 461.50px; height: 381.79px;"><img alt="" src="images/image3.png" style="width: 461.50px; height: 381.79px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c11 c6"><span class="c7 c1"></span></p><p class="c5"><span class="c1 c17">Table 2:</span><span class="c7 c1">&nbsp;Classified expression of genes relative to each contrast. 369/178 genes were up/down regulated in normal sigmoid tissue compared to ulcerative colitis. 520/309 genes were up/down in the normal sigmoid tissue compared to ulcerative colitis. 1337/821 genes were down/up regulated in normal sigmoid tissue compared to normal non-sigmoidal tissue. 1655/661 genes were down-regulated in the ulcerative colitis sigmoid tissue compared to ulcerative colitis non-sigmoidal tissue. 28/25 genes show an increase/decrease in gene expression when changing from normal phenotype to ulcerative colitis or vice versa.</span></p><p class="c5"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 73.33px;"><img alt="" src="images/image1.png" style="width: 624.00px; height: 73.33px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span class="c7 c9">Results:</span></p><p class="c5 c12"><span class="c7 c1">Looking at the first contrast, variable phenotype in sigmoidal tissue, a volcano plot was created mapping the log fold change along the x axis and the significance along the y-axis. The top 4 genes were highlighted based on highest significance, indicated in Figure 2. Those with high absolute value log-fold change were also looked into. </span></p><p class="c5 c6"><span class="c7 c1"></span></p><p class="c5"><span class="c1 c17">Figure 2: </span><span class="c7 c1">Expression fold change for disease phenotype with sigmoidal tissue. Top 4 significant genes were highlighted and labeled.</span></p><p class="c11"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 528.50px; height: 437.49px;"><img alt="" src="images/image4.png" style="width: 528.50px; height: 437.49px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><hr style="page-break-before:always;display:none;"></p><p class="c5"><span class="c1 c17">Figure 3:</span><span class="c7 c1">&nbsp;A volcano plot of the interaction of the disease phenotype on the regional expression. Same as with the contrast for condition with sigmoid tissue, the top 4 significant genes are highlighted in the plot. It is worth noting that the p-values for interaction is lower compared to other contrasts statistics. </span></p><p class="c11"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 525.50px; height: 435.08px;"><img alt="" src="images/image2.png" style="width: 525.50px; height: 435.08px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5 c12"><span class="c7 c1">With this interaction data, a filter was made for those genes with a p-value&lt;0.001 and a threshold log fold change greater than 0.2, these genes are shown in Table 3 below.</span></p><hr style="page-break-before:always;display:none;"><p class="c5 c6"><span class="c7 c1"></span></p><p class="c5"><span class="c1 c17">Table 3: </span><span class="c1">Genes that had significant and high LFC of interaction with phenotype and region. Two genes are unknown in function (</span><span class="c3">A_24_P76362 and A_24_P144543).</span></p><a id="t.ec17a192310f2b1f05176c53066fbfc450f2bbdb"></a><a id="t.0"></a><table class="c25"><tbody><tr class="c15"><td class="c2" colspan="1" rowspan="1"><p class="c14 c6"><span class="c7 c13"></span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c14"><span class="c3">gene</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c14"><span class="c3">LFC_condition_sigmoid</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c14"><span class="c3">LFC_condition_other</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c14"><span class="c3">LFC_tissue_wt</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c14"><span class="c3">LFC_tissue_UC</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c14"><span class="c3">LFC_interaction</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c14"><span class="c3">adj.P.Val</span></p></td></tr><tr class="c15"><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">1</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c14"><span class="c3">CXCL8</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">-0.49142</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">-0.13599</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">0.019351</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">-0.33607</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">0.355423</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">2.82E-06</span></p></td></tr><tr class="c15"><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">2</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c14"><span class="c3">A_24_P76362</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">0.024418</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">-0.20449</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">-0.2906</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">-0.06169</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">-0.22891</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">3.89E-08</span></p></td></tr><tr class="c15"><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">3</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c14"><span class="c3">FMOD</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">0.048445</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">-0.15416</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">-0.28304</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">-0.08044</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">-0.2026</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">0.000193</span></p></td></tr><tr class="c15"><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">4</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c14"><span class="c3">DUOX2.1</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">-0.40503</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">-0.18887</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">0.058714</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">-0.15745</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">0.216165</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">1.08E-05</span></p></td></tr><tr class="c15"><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">5</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c14"><span class="c3">DPP10.AS1</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">0.464421</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">0.214602</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">-0.0899</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">0.159915</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">-0.24982</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">7.99E-07</span></p></td></tr><tr class="c15"><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">6</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c14"><span class="c3">TCN1</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">-0.23333</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">-0.01767</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">0.051143</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">-0.16452</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">0.215658</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">0.000294</span></p></td></tr><tr class="c15"><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">7</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c14"><span class="c3">FAM111B</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">0.18543</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">-0.03451</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">-0.30565</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">-0.08571</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">-0.21994</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">7.86E-05</span></p></td></tr><tr class="c15"><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">8</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c14"><span class="c3">A_24_P144543</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">0.028169</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">0.236376</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">0.092302</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">-0.11591</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">0.208208</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">0.000286</span></p></td></tr><tr class="c15"><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">9</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c14"><span class="c3">UGT2A3</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">0.571</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">0.097747</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">-0.20716</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">0.266097</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">-0.47325</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">0.000233</span></p></td></tr><tr class="c15"><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">10</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c14"><span class="c3">CXCL1</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">-0.50452</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">-0.27757</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">-0.1366</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">-0.36355</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">0.226944</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c10"><span class="c3">2.17E-05</span></p></td></tr></tbody></table><p class="c5 c6"><span class="c7 c1"></span></p><p class="c5"><span class="c7 c9">Conclusions:</span></p><p class="c5 c12"><span class="c7 c1">In this investigation, 4 important genes were found in the sigmoid colon that showed significant difference in expression when comparing the sigmoid colon of an ulcerative colitis patient to one of a normal individual. These genes were: HPDL, ADGRV10, GPR110, and DPP110.AS1. The reason these genes are important is because they were found significant in other papers where the ulcerative colitis patients were compared to normal patients. This helps reaffirm the results obtained in our investigation are valid as they are shown to be true by looking at previous studies. Thus, these genes are essentially controls in our project as they serve as a reference point to show that our analysis was done correctly. It is worth noting that these genes are typically expressed higher in the sigmoid colon in comparison to other parts of the colon, regardless of whether the patient has ulcerative colitis or not (data not shown). Once this was observed, we can now proceed to look at some other genes that are shown to have a significant interaction effect, which are shown in Table 3.</span></p><p class="c5 c12"><span class="c7 c1">Although 10 genes were shown to have interactions in Table 3, specifically 4 genes are of tremendous interest in regards to our study as they are directly related to inflammatory processes related to ulcerative colitis. These genes are: UGT2A3, CXCL1, CXCL8, and DPP10.AS1. UGT2A3, which is known to play a significant role in metabolism--specifically of drugs, is of interest because alteration to the expression of this gene can detrimentally affect the extent to which drugs are metabolized. By looking at Table 3, it is observed that the UGT2A3 gene has a positive value for LFC_condition_sigmoid and LFC_condition_other, which means that it is downregulated in ulcerative colitis tissues relative to normal patients. This implies that ulcerative colitis patients are not able to effectively break down drugs effectively, which can lead to the buildup of toxins in the intestines. These toxins could alter cellular function in the intestines leading to inflammation being triggered by the immune response, which ultimately leads to the ulcerative colitis phenotype in patients. </span></p><p class="c5 c12"><span class="c7 c1">Next, we move onto the CXCL1 and CXCL8 genes as they are directly involved in the inflammatory response by helping macrophages release chemokines that trigger inflammatory processes. By looking at Table 3, it is interesting to see that we have significantly more &nbsp;negative values for LFC_condition_sigmoid in comparison to LFC_condition_other. This means that the CXCL1 and CXCL8 genes are upregulated in ulcerative colitis patients, and leads to an overexpression of the chemokines that result in an exacerbated inflammatory response. Looking at the disease itself, this makes sense with what has been stated in previous literature as excessive unnecessary inflammation is common in many cases of ulcerative colitis. </span></p><p class="c5 c12"><span class="c7 c1">Now, we move on to discussing DPP10.AS1, which is a gene that has a connection to the development of colon cancer. Given that LFC_condition_sigmoid has positive value for this gene, it is implied that this gene is downregulated in ulcerative colitis patients in comparison to normal patients. What further makes this interesting is that since this gene is linked to colon cancer, it is likely that is a tumor suppressor gene whose function is lowered leading to the manifestation of ulcerative colitis, which ultimately progresses to cancer as IBD is one of the leading causes of colon cancer world-wide. Lastly, it is worth noting that A_24_P144543 and A_24_P76362 also possess significant results as these genes appear to very downregulated in ulcerative colitis patients in comparison to normal patients given the fact that these genes have positive values in the LFC_condition_sigmoid columns. Currently much knowledge of these genes is currently unknown, but future investigations of these genes may provide further insight to the functions of A_24_P144543 and A_24_P76362 and their connection to ulcerative colitis. </span></p><p class="c5 c12"><span class="c7 c1">A limitation to this data is how the factorial matrix was created. Because the tissue types were only sigmoid tissue compared to other tissue, significantly expressed genes in a region such as the ascending colon relative to the sigmoid region cannot be analyzed. Future models could implement different base tissue such as setting the sigmoid region to ascending colon in this model. Additionally, the condition for ulcerative colitis is identified as a dummy variable. Someone who has more severe symptoms such as more frequent bowel movements is treated the same in this model as someone with less. As such, variable expressivity isn&rsquo;t accounted for in the linear model created. Future remedies for this seem difficult, as a term such as number of movements is subject to error.</span></p><p class="c5 c12"><span class="c7 c1">For future investigations, we must take other factors into account that can drastically affect our results in future investigations. One of these factors is taking other kinds of IBD into account, such as Crohn&rsquo;s disease, in order to better generalize our results to all kinds of IBD. The reason being is that ulcerative colitis and Crohn&rsquo;s disease are different to an extent to which parts of the gastrointestinal tract they affect, and as a result both IBDs must be looked at in order to identify generalizable results that apply to both IBDs. Additionally, age of diagnosis must be taken into account for future investigations. The reason being is that IBD does not manifest at birth, but rather typically appears randomly in people&rsquo;s lifetimes. This implies that for cases of IBD have variable penetrance in regards to age of manifestation, which may be due to the result of how mutated a particular gene may be. Lastly, For example, more mutations may lead to earlier penetrance of the IBD phenotype, while less mutations may lead to later penetrance of this phenotype. Lastly, environmental factors, such as stress, diet, place of living, and etc, must also be taken into account as previous literature has shown that IBD is not necessarily the product of gene mutations, but may be rather a product of gene-environment interaction. Thus, in the end, this investigation only brings us a small step closer to a cure as there is much we still need to discover in regards to inflammatory bowel disease. </span></p><p class="c5 c6"><span class="c7 c1"></span></p><p class="c5"><span class="c7 c9">Contributions:</span></p><p class="c5"><span class="c1 c17">Conceptualization--</span><span class="c7 c1">&nbsp;Chandershekhar was the one who primarily undertook the endeavor of producing the ideas, goals, and gathering background information of this investigation.</span></p><p class="c5 c6"><span class="c7 c1"></span></p><p class="c5"><span class="c1 c17">Data Curation--</span><span class="c7 c1">&nbsp;Steven and Chandershekhar both assisted in helping prepare this data for the purposes of this investigation and for later use.</span></p><p class="c5 c6"><span class="c7 c1"></span></p><p class="c5"><span class="c1 c17">Formal Analysis--</span><span class="c7 c1">&nbsp;Steven and Chandershekhar both undertook the endeavor of producing the code necessary for analysis of this investigation. </span></p><p class="c5 c6"><span class="c7 c1"></span></p><p class="c5"><span class="c1 c17">Methodology-- </span><span class="c7 c1">Steven and Chandershekhar both assisted in creating the methods needed to proceed with this investigation.</span></p><p class="c5 c6"><span class="c7 c1"></span></p><p class="c5"><span class="c1 c17">Funding Acquisition--</span><span class="c7 c1">&nbsp;There really was no extra funding needed to carry out this investigation. All resources were provided by the University of Texas at Austin and Dr. Cenik&rsquo;s teaching team.</span></p><p class="c5 c6"><span class="c7 c1"></span></p><p class="c5"><span class="c1 c17">Investigation--</span><span class="c7 c1">&nbsp;The investigation was supervised and conducted by both Steven and Chandershekhar.</span></p><p class="c5 c6"><span class="c7 c1"></span></p><p class="c5"><span class="c1 c17">Project Administration--</span><span class="c7 c1">&nbsp;The investigation was managed and coordinated by both Steven and Chandershekhar.</span></p><p class="c5 c6"><span class="c7 c1"></span></p><p class="c5"><span class="c1 c17">Resources--</span><span class="c7 c1">&nbsp;Chandershekhar and Steven gathered their resources using the Gene Expression Omnibus (GEO) database, and attained much of their other resources from Dr. Cenik&rsquo;s class and teaching team.</span></p><p class="c5 c6"><span class="c7 c1"></span></p><p class="c5"><span class="c1 c17">Software--</span><span class="c7 c1">&nbsp;Chandershekhar and Steven attained much of their software from R packages online, and Dr. Cenik&rsquo;s teaching team. The software used for this investigation was R.</span></p><p class="c5 c6"><span class="c7 c1"></span></p><p class="c5"><span class="c1 c17">Visualization--</span><span class="c7 c1">&nbsp;Chandershekhar and Steven both assisted in &nbsp;creating the data and preparing it for visualization through the powerpoint presentation.</span></p><p class="c5 c6"><span class="c7 c1"></span></p><p class="c5"><span class="c1 c17">Supervision-- </span><span class="c7 c1">Chandershekhar and Steven received supervision from Dr. Cenik&rsquo;s teaching team as they helped provide the necessary mentorship to help make this project a success. They did this by providing the research team with constructive critiques to help guide their project onto the right path.</span></p><p class="c5 c6"><span class="c7 c1"></span></p><p class="c5"><span class="c1 c17">Validation-- </span><span class="c7 c1">Chandershekhar and Steven both helped verify the replicability of the results produced in this investigation.</span></p><p class="c5 c6"><span class="c7 c1"></span></p><p class="c5"><span class="c1 c17">Writing -original draft-- </span><span class="c7 c1">&nbsp;Chandershekhar and Steven both assisted in preparing the initial draft and presentation for this investigation.</span></p><p class="c5 c6"><span class="c7 c1"></span></p><p class="c5"><span class="c1 c17">Writing - review &amp; editing--</span><span class="c7 c1">&nbsp; Chandershekhar and Steven both assisted in finalizing and proofing the final overall presentation and paper. </span></p><p class="c5 c6"><span class="c7 c1"></span></p><p class="c5 c6"><span class="c7 c1"></span></p><hr style="page-break-before:always;display:none;"><p class="c0 c6"><span class="c7 c9"></span></p><p class="c19"><span class="c7 c9">References:</span></p><p class="c22 c6"><span class="c4"></span></p><p class="c22"><span class="c1">Mackenzie, Peter I., et al. &ldquo;The Novel UDP Glycosyltransferase 3A2: Cloning, Catalytic Properties, and Tissue Distribution.&rdquo; </span><span class="c1">Molecular Pharmacology</span><span class="c7 c1">, vol. 79, no. 3, 2010, pp. 472&ndash;478., doi:10.1124/mol.110.069336.</span></p><p class="c22"><span class="c1">Mirza, Aashiq H, et al. &ldquo;Transcriptomic Landscape of LncRNAs in Inflammatory Bowel Disease.&rdquo; </span><span class="c7 c1">Genome Medicine, vol. 7, no. 1, 2015, doi:10.1186/s13073-015-0162-2.</span></p><p class="c22"><span class="c1">Noble, C L, et al. &ldquo;Regional Variation in Gene Expression in the Healthy Colon Is Dysregulated in Ulcerative Colitis.&rdquo; Gut, vol. 57, no. 10, 2008, pp. 1398&ndash;1405., doi:10.1136/gut.2008.148395.</span></p><p class="c22"><span class="c7 c1">Ritchie, ME, Phipson, B, Wu, D, Hu, Y, Law, CW, Shi, W, and Smyth, GK (2015).limma powers differential expression analyses for RNA-sequencing and microarray studies.Nucleic Acids Research43 (7), e47.</span></p><p class="c22"><span class="c1">Phipson, B, Lee, S, Majewski, IJ, Alexander, WS, and Smyth, GK (2016). Robust Hyperparameter estimation protects against hypervariable genes and </span><span class="c7 c1">improves power to detect differential expression.Annals of Applied Statistics 10(2), 946&ndash;963</span></p><p class="c22"><span class="c1">Law, CW, Chen, Y, Shi, W, and Smyth, GK (2014). Voom: precision weights unlock linear model analysis tools for RNA-seq read counts.Genome Biology15, R29</span></p><p class="c19 c6"><span class="c7 c9"></span></p><p class="c19 c6"><span class="c7 c9"></span></p><p class="c19 c6"><span class="c7 c9"></span></p><p class="c19 c6"><span class="c7 c9"></span></p><p class="c19 c6"><span class="c7 c9"></span></p><p class="c19 c6"><span class="c7 c9"></span></p><p class="c19 c6"><span class="c7 c9"></span></p><p class="c19 c6"><span class="c7 c9"></span></p><p class="c19 c6"><span class="c7 c9"></span></p><p class="c19 c6"><span class="c7 c9"></span></p><p class="c6 c19"><span class="c7 c9"></span></p><p class="c19 c6"><span class="c7 c9"></span></p><p class="c19 c6"><span class="c7 c9"></span></p><hr style="page-break-before:always;display:none;"><p class="c5 c6"><span class="c7 c9"></span></p><p class="c5"><span class="c9">Code:</span></p><p class="c0"><span class="c4">library(rjson)</span></p><p class="c0"><span class="c4">library(BiocManager)</span></p><p class="c0"><span class="c4">library(GEOquery)</span></p><p class="c0"><span class="c4">library(pheatmap)</span></p><p class="c0"><span class="c4">library(DESeq2)</span></p><p class="c0"><span class="c4">library(limma)</span></p><p class="c0"><span class="c4">library(tidyverse)</span></p><p class="c0"><span class="c4"># updateR()</span></p><p class="c0 c6"><span class="c4"></span></p><p class="c0"><span class="c4">#Setting up for limma, the expression matrix, the feature data, and the phenotype data</span></p><p class="c0"><span class="c4">BiocManager::install(&quot;limma&quot;)</span></p><p class="c0 c6"><span class="c4"></span></p><p class="c0"><span class="c4">gds858 &lt;- getGEO(&#39;GDS3268&#39;, destdir=&quot;.&quot;) #expression</span></p><p class="c0"><span class="c4">agilent_source = getGEO(&quot;GPL1708&quot;, destdir=&quot;.&quot;) #feature matrix of the assay, includes genes and descriptions</span></p><p class="c0"><span class="c4">tab = Table(gds858)</span></p><p class="c0"><span class="c4">tab_gds858 = tab %&gt;% select(3:length(tab[1,]))</span></p><p class="c0"><span class="c4">gds858_logcounts = as.matrix(tab_gds858)</span></p><p class="c0"><span class="c4">rownames(gds858_logcounts)=make.names(tab[,2],unique=TRUE)</span></p><p class="c0"><span class="c4">rownames(gds858_logcounts)</span></p><p class="c0"><span class="c4">gds858_logcounts=na.omit(gds858_logcounts)</span></p><p class="c0 c6"><span class="c4"></span></p><p class="c0"><span class="c4">agilentmatrix=data.frame(agilent_source@dataTable@table)</span></p><p class="c0"><span class="c4">rownames(agilentmatrix)=make.names(agilentmatrix$SPOT_ID,unique=TRUE)</span></p><p class="c0"><span class="c4"># fagilent = agilentmatrix %&gt;% select(GENE_SYMBOL,GENE_NAME,DESCRIPTION)</span></p><p class="c0"><span class="c4"># reorder_agilent &lt;- match(rownames(agilentmatrix),rownames(gds858_logcounts))</span></p><p class="c0"><span class="c4"># reorder_agilent = na.omit(reorder_agilent)</span></p><p class="c0"><span class="c4"># identical(rownames(agilentmatrix),rownames(gds858_logcounts))</span></p><p class="c0"><span class="c4"># agilentmatrix=na.omit(agilentmatrix)</span></p><p class="c0 c6"><span class="c4"></span></p><p class="c0 c6"><span class="c4"></span></p><p class="c0 c6"><span class="c4"></span></p><p class="c0"><span class="c4"># agilentmatrix = agilentmatrix %&gt;% filter(rownames(agilentmatrix) %in% rownames(gds858_logcounts))</span></p><p class="c0"><span class="c4">gds858_metadata= gds858@dataTable@columns</span></p><p class="c0"><span class="c4">gds858_metadata=column_to_rownames(gds858_metadata,&quot;sample&quot;) #phenotype</span></p><p class="c0 c6"><span class="c4"></span></p><p class="c0"><span class="c4">#This is adjusting the phenotype data to match the expression data</span></p><p class="c0"><span class="c4">reorder_idx &lt;- match(rownames(gds858_metadata),colnames(gds858_logcounts))</span></p><p class="c0"><span class="c4">prepared_gds858 = gds858_metadata[reorder_idx,]</span></p><p class="c0"><span class="c4">prepared_gds858 = data.frame(prepared_gds858)</span></p><p class="c0 c6"><span class="c4"></span></p><p class="c0"><span class="c4">#column names were adjusted to have better labels, as well as group the other sigmoid regions into one</span></p><p class="c0"><span class="c4">prepared_gds858 = prepared_gds858 %&gt;% mutate(sigmoid=</span></p><p class="c0"><span class="c4">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;case_when(</span></p><p class="c0"><span class="c4">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;tissue==&quot;sigmoid colon&quot;~&quot;sigmoid&quot;,</span></p><p class="c0"><span class="c4">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;TRUE ~ &quot;other&quot;</span></p><p class="c0"><span class="c4">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;))</span></p><p class="c0"><span class="c4">prepared_gds858 = prepared_gds858 %&gt;% mutate(disease.state=</span></p><p class="c0"><span class="c4">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;case_when(</span></p><p class="c0"><span class="c4">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;disease.state==&quot;ulcerative colitis&quot;~&quot;UC&quot;,</span></p><p class="c0"><span class="c4">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;TRUE ~ &quot;wt&quot;</span></p><p class="c0"><span class="c4">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;))</span></p><p class="c0"><span class="c4">rownames(prepared_gds858)=colnames(gds858_logcounts)</span></p><p class="c0 c6"><span class="c4"></span></p><p class="c0 c6"><span class="c4"></span></p><p class="c0"><span class="c4"># dim(gds858_logcounts)</span></p><p class="c0"><span class="c4">prepared_gds858 = sapply(prepared_gds858,as.character)</span></p><p class="c0"><span class="c4"># sapply(prepared_gds858,class)</span></p><p class="c0"><span class="c4">prepared_gds858=data.frame(prepared_gds858)</span></p><p class="c0"><span class="c4">rownames(prepared_gds858)=colnames(gds858_logcounts)</span></p><p class="c0 c6"><span class="c4"></span></p><p class="c0 c6"><span class="c4"></span></p><p class="c0"><span class="c4">#There was trouble adding the agilent_microarray_matrix onto the feature data for the ExpressionSet</span></p><p class="c0"><span class="c4">#object, so if gene names were different than what was in the limma model ID!=gene, then it was</span></p><p class="c0"><span class="c4">#checked manually.</span></p><p class="c0 c6"><span class="c4"></span></p><p class="c0"><span class="c4">eset =ExpressionSet(gds858_logcounts,phenoData=AnnotatedDataFrame(prepared_gds858))</span></p><p class="c0"><span class="c4">plotDensities(eset,legend=FALSE,main=&quot;Distribution of Log Ratio MicroArray Data&quot;)</span></p><p class="c0 c6"><span class="c4"></span></p><p class="c0 c6"><span class="c4"></span></p><p class="c0"><span class="c4">#model matrix was created with the 4 parts in the model</span></p><p class="c0"><span class="c4">group &lt;- with(pData(eset), trimws(paste(disease.state, sigmoid, sep = &quot;.&quot;)))</span></p><p class="c0"><span class="c4">group &lt;- factor(group)</span></p><p class="c0"><span class="c4">group</span></p><p class="c0"><span class="c4">design &lt;- model.matrix(~0 + group)</span></p><p class="c0"><span class="c4">colnames(design) &lt;- levels(group)</span></p><p class="c0"><span class="c4">colSums(design)</span></p><p class="c0 c6"><span class="c4"></span></p><p class="c0"><span class="c4">#Contrasts were made among the two factors as well as the interaction</span></p><p class="c0"><span class="c4">cm &lt;- makeContrasts(condition_sigmoid = wt.sigmoid - UC.sigmoid,</span></p><p class="c0"><span class="c4">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;condition_other = wt.other - UC.other,</span></p><p class="c0"><span class="c4">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;tissue_wt = wt.other - wt.sigmoid,</span></p><p class="c0"><span class="c4">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;tissue_UC = UC.other - UC.sigmoid,</span></p><p class="c0"><span class="c4">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;interaction = (wt.other - wt.sigmoid) - (UC.other - UC.sigmoid),</span></p><p class="c0"><span class="c4">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;levels = design)</span></p><p class="c0"><span class="c4">fit &lt;- lmFit(eset, design)</span></p><p class="c0 c6"><span class="c4"></span></p><p class="c0"><span class="c4">fit2 &lt;- contrasts.fit(fit, contrasts = cm)</span></p><p class="c0 c6"><span class="c4"></span></p><p class="c0"><span class="c4"># Calculate the t-statistics for the contrasts</span></p><p class="c0"><span class="c4">fit2 &lt;- eBayes(fit2)</span></p><p class="c0 c6"><span class="c4"></span></p><p class="c0"><span class="c4"># Summarize results</span></p><p class="c0"><span class="c4">results &lt;- decideTests(fit2)</span></p><p class="c0"><span class="c4">results &lt;- classifyTestsF(fit2, p.value=0.0001)</span></p><p class="c0"><span class="c4"># vennDiagram(results,include=c(&quot;up&quot;))</span></p><p class="c0"><span class="c4">#understanding the diagram by looking at contrasts,</span></p><p class="c0"><span class="c4">summary(results)</span></p><p class="c0"><span class="c4">cm</span></p><p class="c0 c6"><span class="c4"></span></p><p class="c0"><span class="c4">stats &lt;- topTable(fit2, number = nrow(fit2), sort.by = &quot;none&quot;,adjust.method=&#39;BH&#39; )</span></p><p class="c0"><span class="c4"># stats %&gt;% order_by(adj.P.Val)</span></p><p class="c0"><span class="c4">hist(stats[, &quot;P.Value&quot;])</span></p><p class="c0"><span class="c4">#Histogram was plotted to make sure the p values distributions reflect typical limma models</span></p><p class="c0"><span class="c4">#Left skewed is normal</span></p><p class="c0 c6"><span class="c4"></span></p><p class="c0"><span class="c4">volcanoplot(fit2, highlight = 4, names = rownames(fit2$p.value),coef=&quot;interaction&quot;,main=&quot;Differences in Interaction of normal/UC on sigmoid tissue&quot;)</span></p><p class="c0"><span class="c4">volcanoplot(fit2, highlight = 4, names = rownames(fit2$p.value),coef=&quot;condition_sigmoid&quot;,main=&quot;Expression Fold Change for normal and UC in Sigmoid&quot;)</span></p><p class="c0 c6"><span class="c4"></span></p><p class="c0 c6"><span class="c4"></span></p><p class="c0"><span class="c4">#looked at gene expression in other tissue, omitted from paper</span></p><p class="c0"><span class="c4">volcanoplot(fit2, highlight = 4, names = rownames(fit2$p.value),coef=&quot;condition_other&quot;,main=&quot;Expression Fold Change for normal and UC in non-sigmoid&quot;)</span></p><p class="c0 c6"><span class="c4"></span></p><p class="c0 c6"><span class="c4"></span></p><p class="c0 c6"><span class="c4"></span></p><p class="c0"><span class="c4">#These two volcano plots show which genes are expressed more in which tissue.</span></p><p class="c0"><span class="c4">#For the purposes of the paper, they were excluded, but there are high significance values</span></p><p class="c0"><span class="c4">#as well as nice plots.</span></p><p class="c0 c6"><span class="c4"></span></p><p class="c0"><span class="c4">volcanoplot(fit2, highlight = 4, names = rownames(fit2$p.value),coef=&quot;tissue_wt&quot;,main=&quot;Expression Fold Change for regions in WT&quot;)</span></p><p class="c0"><span class="c4">volcanoplot(fit2, highlight = 4, names = rownames(fit2$p.value),coef=&quot;tissue_UC&quot;,main=&quot;Expression Fold Change for regions in UC&quot;)</span></p><p class="c0 c6"><span class="c4"></span></p><p class="c0"><span class="c4">stats = rownames_to_column(stats,var=&quot;gene&quot;)</span></p><p class="c0 c6"><span class="c4"></span></p><p class="c0"><span class="c4">#analysis was done on the interaction term. Here I filtered genes that had p value&lt;0.001,</span></p><p class="c0"><span class="c4">#and these genes had a coefficient higher than 0.2.</span></p><p class="c0"><span class="c4">interaction_stats = stats %&gt;% filter(adj.P.Val&lt;0.001,abs(interaction)&gt;0.2)</span></p><p class="c0"><span class="c4">interaction_stats</span></p><p class="c0 c6"><span class="c4"></span></p><p class="c0"><span class="c4">#CXCL1 and CXCL8 are chemokines, which is interesting.</span></p><p class="c0"><span class="c4">#UGT3A2 is a detoxifying protein that changes lipophillic things to liquid. This may explain damage,</span></p><p class="c0"><span class="c4">#though unknown for now.</span></p><p class="c0"><span class="c4"># transformed_stats %&gt;% mutate(average=average())</span></p><p class="c0"><span class="c4">write.csv(interaction_stats,&quot;finish_data2.csv&quot;)</span></p><p class="c0"><span class="c4">write.csv(stats,&quot;whole_stats.csv&quot;)</span></p><p class="c0 c6"><span class="c4"></span></p><p class="c0 c6"><span class="c4"></span></p><p class="c0 c6"><span class="c4"></span></p><p class="c0 c6"><span class="c4"></span></p><p class="c0 c6"><span class="c4"></span></p><p class="c0 c6"><span class="c4"></span></p><p class="c0 c6"><span class="c4"></span></p><p class="c0 c6"><span class="c4"></span></p><p class="c0 c6"><span class="c4"></span></p><p class="c0"><span class="c4"># cor_normalized = cor(gds858_rawcounts_,y=NULL, use = &quot;everything&quot;, method = &quot;spearman&quot;)</span></p><p class="c0"><span class="c4"># pheatmap(cor_normalized,na.rm=TRUE)</span></p><p class="c0 c6"><span class="c4"></span></p><p class="c0 c6"><span class="c4"></span></p><p class="c0 c6"><span class="c4"></span></p><p class="c0 c6"><span class="c4"></span></p><p class="c0 c6"><span class="c4"></span></p><p class="c0 c6"><span class="c4"></span></p><p class="c0"><span class="c4"># dds_alls &lt;- DESeqDataSetFromMatrix(countData = gds858_rawcounts_,</span></p><p class="c0"><span class="c4"># &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;colData = prepared_gds858,</span></p><p class="c0"><span class="c4"># &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;design = ~condition + region)</span></p><p class="c0"><span class="c4"># class(dds_alls)</span></p><p class="c0"><span class="c4"># class(deseq_colon)</span></p><p class="c0"><span class="c4"># deseq_colon = DESeq(dds_alls)</span></p><p class="c0"><span class="c4"># keep &lt;- rowMeans(counts(deseq_colon)) &gt;= 500</span></p><p class="c0"><span class="c4"># deseq_colon = deseq_colon[keep,]</span></p><p class="c0"><span class="c4"># deseq_colon$region= relevel(deseq_colon$region,ref=&quot;sigmoid colon&quot;)</span></p><p class="c0"><span class="c4"># deseq_colon$condition= relevel(deseq_colon$condition,ref=&quot;normal&quot;)</span></p><p class="c0"><span class="c4">#</span></p><p class="c0"><span class="c4"># #outlier detection</span></p><p class="c0"><span class="c4"># BiocManager::install(&quot;apeglm&quot;)</span></p><p class="c0"><span class="c4"># resLFC &lt;- lfcShrink(deseq_colon, coef=&quot;condition_ulcerative.colitis_vs_normal&quot;,type=&quot;apeglm&quot;)</span></p><p class="c0"><span class="c4"># plotDispEsts()</span></p><p class="c0"><span class="c4"># plotDispEsts(resLFC)</span></p><p class="c0"><span class="c4"># plotMA(resLFC, ylim=c(-2,2))</span></p><p class="c0"><span class="c4"># plotMA(dds_results,ylim=c(-2,2))</span></p><p class="c0"><span class="c4">#</span></p><p class="c0"><span class="c4"># #vst was chosen for log transform and normalization</span></p><p class="c0"><span class="c4"># vsd_all &lt;- vst(deseq_colon, blind = TRUE)</span></p><p class="c0"><span class="c4"># plotPCA(vsd_all, intgroup=c(&quot;condition&quot;, &quot;region&quot;))</span></p><p class="c0"><span class="c4"># plotPCA(vsd_all, intgroup=c(&quot;condition&quot;))</span></p><p class="c0"><span class="c4"># plotPCA(vsd_all, intgroup=c(&quot;region&quot;))</span></p><p class="c0"><span class="c4"># # resultsNames(deseq_colon)</span></p><p class="c0"><span class="c4"># vsst_colon = vst(deseq_colon, blind=TRUE)</span></p><p class="c0"><span class="c4"># dds_results = results(deseq_colon,contrast=c(&quot;condition&quot;,&quot;ulcerative.colitis&quot;,&quot;normal&quot;),</span></p><p class="c0"><span class="c4"># &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;alpha=0.05, lfcThreshold=0.32)</span></p><p class="c0"><span class="c4"># dds_results &lt;- lfcShrink(deseq_colon,</span></p><p class="c0"><span class="c4"># &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;contrast=c(&quot;condition&quot;,&quot;ulcerative.colitis&quot;,&quot;normal&quot;),</span></p><p class="c0"><span class="c4"># &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;res=dds_results)</span></p><p class="c0"><span class="c4"># dds_data_frame = data.frame(dds_results)</span></p><p class="c0"><span class="c4"># dds_final &lt;- subset(dds_data_frame, pvalue &lt; 0.05)</span></p><p class="c0"><span class="c4"># write.csv(dds_final,&quot;DESeq_gene_results&quot;)</span></p><p class="c0"><span class="c4"># vst_ordered = vsd_all[order(vsd_all$pvalue),]</span></p><p class="c0"><span class="c4">#</span></p><p class="c0"><span class="c4">#</span></p><p class="c0"><span class="c4"># dds_condition_only = DESeqDataSetFromMatrix(countData = gds858_rawcounts_,</span></p><p class="c0"><span class="c4"># &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;colData = prepared_gds858,</span></p><p class="c0"><span class="c4"># &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;design = ~condition)</span></p><p class="c0"><span class="c4"># dds_condition_only$condition= relevel(dds_condition_only$condition,ref=&quot;normal&quot;)</span></p><p class="c0"><span class="c4"># dds_condition_only = DESeq(dds_condition_only)</span></p><p class="c0"><span class="c4"># plotDispEsts(dds_condition_only, main = &quot;condition only DispEsts&quot;)</span></p><p class="c0"><span class="c4"># dds_region_res = vst(dds_region_only,blind=TRUE)</span></p><p class="c0"><span class="c4"># plotPCA(dds_region_res, intgroup=&quot;region&quot;)</span></p><p class="c0"><span class="c4">#</span></p><p class="c0"><span class="c4"># dds_region_only = DESeqDataSetFromMatrix(countData = gds858_rawcounts_,</span></p><p class="c0"><span class="c4"># &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;colData = prepared_gds858,</span></p><p class="c0"><span class="c4"># &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;design = ~region)</span></p><p class="c0"><span class="c4"># dds_region_only$region= relevel(dds_region_only$region,ref=&quot;sigmoid colon&quot;)</span></p><p class="c0"><span class="c4"># dds_region_only = DESeq(dds_region_only)</span></p><p class="c0"><span class="c4"># plotDispEsts(dds_region_only,main=&quot;region only DispEsts&quot;)</span></p><p class="c0"><span class="c4"># dds_region_res = vst(dds_region_only,blind=TRUE)</span></p><p class="c0"><span class="c4"># plotPCA(dds_region_res, intgroup=&quot;region&quot;)</span></p><p class="c0"><span class="c4">#</span></p><p class="c0"><span class="c4">#</span></p><p class="c0"><span class="c4"># library(&quot;RColorBrewer&quot;)</span></p><p class="c0"><span class="c4"># library(&quot;pheatmap&quot;)</span></p><p class="c0"><span class="c4"># sampleDists &lt;- dist(t(assay(vsd_all)))</span></p><p class="c0"><span class="c4"># sampleDistMatrix &lt;- as.matrix(sampleDists)</span></p><p class="c0"><span class="c4"># rownames(sampleDistMatrix) &lt;- paste(vsd_all$condition, vsd_all$region)</span></p><p class="c0"><span class="c4"># colnames(sampleDistMatrix) &lt;- NULL</span></p><p class="c0"><span class="c4"># colors &lt;- colorRampPalette( rev(brewer.pal(9, &quot;Blues&quot;)) )(255)</span></p><p class="c0"><span class="c4"># window(width=12,height=10)</span></p><p class="c0"><span class="c4"># pheatmap(sampleDistMatrix,</span></p><p class="c0"><span class="c4"># &nbsp; &nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;clustering_distance_rows=sampleDists,</span></p><p class="c0"><span class="c4"># &nbsp; &nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;clustering_distance_cols=sampleDists,</span></p><p class="c0"><span class="c4"># &nbsp; &nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;col=colors,show_rownames = F)</span></p><p class="c0"><span class="c4">#</span></p><p class="c0"><span class="c4">#</span></p><p class="c0 c6"><span class="c4"></span></p><p class="c0 c6"><span class="c4"></span></p></body></html>